src/api/config/routes/webui_routes.rb

```diff
@@ -389,6 +389,9 @@
     ### /apidocs
     get 'apidocs', to: redirect('/apidocs/index')
     get 'apidocs/(index)' => 'webui/apidocs#index', as: 'apidocs_index'
+    controller 'webui/badge' do
+      get 'badge/:project/:package.svg' => :show, constraints: cons, defaults: { format: 'svg' }
+    end
```

Comment:
```suggestion

    resource :badge, controller: 'webui/badge', only: [], constraints: cons do
      get ':project/:package.:format' => :show, constraints: ->(request) { request.format == :svg }
    end
```
I would change from using a `controller` block to a `resource` block, like suggested. We use `resource` or `resources` whenever is possible. What do you think?